frontend/src/components/ScheduledEventsTableRow/ScheduledEventsTableRow.tsx

```diff
@@ -12,7 +17,52 @@ export interface ScheduledEventsTableRowProps {
   start_date: string;
 }
 
+interface RunInstancePayload {
+  start_at: Moment;
+  status: string;
+}
+
+const BASEPATH = api.routes.VIEW_SCHEDULED_EVENTS;
+
+const createRunInstance = async (
+  scheduleId: number | undefined,
+  data: RunInstancePayload,
+): Promise<any> => {
```

Comment:
Don't you know the shape this promise's return is going to take? Give it a type ... no `any`

Reply:
Also updated